### PR TITLE
2412 enhance integration tests

### LINKED ISSUE #2412: Cover Features in Integration Tests
__What should we do?__

The following new features are not covered in the [integration tests](https://github.com/camunda/camunda-modeler/blob/master/docs/.project/INTEGRATION_TEST.md):

* new form editor features (new form fields)
* execution platform
* linting

__Why should we do it?__

We need to ensure that these essential features are working in every release.

### PR DESCRIPTION
**Pull request by MaxTru**
closes #2412

### CODE REVIEW

File: docs/.project/INTEGRATION_TEST.md [hunk 1]
```diff
@@ -82,10 +82,14 @@ Based on the [test diagram](./test.bpmn.png):
 
 * [ ] create a new Camunda Form
 * [ ] build [this form](./test.form.png) from scratch
+* [ ] add a regular expression (`^CAM-[0-9]+$`) to the invoice number field
 * [ ] save file on disk as `test.form`
 * [ ] file imports correctly after save
-* [ ] add a `required` validation to the creditor field
-* [ ] add a regular expression (`^C-[0-9]+$`) to the invoice number field
+* [ ] Set the **Execution Platform Version** to `Camunda Cloud 1.0`
+    * [ ] 4 errors are shown: `Text`, `Number`, `Radio`, and `Text` are not supported
```
Comment (philippfromme) [inline docs/.project/INTEGRATION_TEST.md:89]:
```suggestion
    * [ ] 4 errors are shown: `Text`, `Number`, `Checkbox`, and `Radio` are not supported
```

**Review by philippfromme — CHANGES_REQUESTED** [review]
Got one remark.

File: docs/.project/INTEGRATION_TEST.md (hunk 1 above)
Comment (MaxTru) [reply docs/.project/INTEGRATION_TEST.md]: Corrected

Comment (MaxTru) [reply docs/.project/INTEGRATION_TEST.md]: Thanks

**Review by philippfromme — APPROVED** [review]
👏🏻

**Pull request merged**